### DISCUSSION TITLE
How to use CSS helper in mat-buttons?

### DISCUSSION THREAD

**Discussion by rahulpd101** [ng-matero/ng-matero#591] [Q&A]
I am trying to give left margin to a button but it is not applying on the button.
I am using this syntax ml-10 (marign-left = 10)
Please help.

**Comment by nzbin** [comment]
Please check the available CSS helpers.
https://ng-matero.github.io/ng-matero/utilities/css-helpers

Indeed, you can modify or extend the CSS helpers by yourself.
https://github.com/ng-matero/ng-matero/blob/main/src/styles/helpers/_variables.scss
